pfcli/deployment.py

```diff
@@ -198,16 +230,21 @@ def list(
     from_oldest: bool = typer.Option(
         False, "--from-oldest", help="Show oldest deployments first"
     ),
+    all: bool = typer.Option(False, "--all", help="Show all deployments in org"),
```

Comment:
I suggest using an option name other than `all`. Maybe `org`?
```suggestion
    org: bool = typer.Option(False, "--org", help="Show all deployments in org"),
```